src/main/java/no/nav/pto/veilarbportefolje/elastic/ElasticQueryBuilder.java

```diff
@@ -166,6 +184,25 @@ private static void defaultSort(String sortField, SearchSourceBuilder searchSour
         }
     }
 
+    static void sorterSisteEndringTidspunkt(SearchSourceBuilder builder, SortOrder order, Filtervalg filtervalg) {
+        String expresion = null;
+        if(filtervalg.sisteEndringKategori.size() == 1) {
+            expresion = "doc['siste_endringer." + filtervalg.sisteEndringKategori.get(0).toLowerCase() + ".tidspunkt']?.value.getMillis()";
```

Comment:
I think we should consider some refactoring after this release, there are few places where we are making this elasticseach key siste_endringer." + filtervalg.sisteEndringKategori.get(0).toLowerCase()... I dont like that we have this toLowerCase in few places. Maybe some mapping between  sisteEndringKategori and elasticsearch siste_endringer object would be cleaner.